### ISSUE TITLE
Share with internal FAH donors?

### ISSUE THREAD

**Issue by cxhernandez** [rmcgibbo/openmm-bench#2]
They could give us plenty of benchmarking stats.

**Comment by rmcgibbo** [comment]
Sounds like a good idea. We will probably have to make the docs better. Also currently it doesn't work on windows. I'm trying to fix that, but it might not be quick. Do any of the internal people run linux?

**Comment by cxhernandez** [comment]
A handful do– but, yeah, a strong majority run Windows.

**Comment by rmcgibbo** [comment]
It also requires having a working compiler toolchain, which is nontrivial on windows.

**Comment by cxhernandez** [comment]
Yeah... that's going to be a issue for most of them...

**Comment by rmcgibbo** [comment]
I think we'll be able to get something useful for OpenMM even without having totally complete coverage of every possible GPU/platform, so w/e.

**Comment by cxhernandez** [comment]
True. I just imagine they would be curious/interested in participating.
